Userland/Libraries/LibWeb/XHR/FormData.h

```diff
@@ -39,6 +39,7 @@ class FormData : public Bindings::PlatformObject {
 private:
     explicit FormData(JS::Realm&, HashMap<DeprecatedString, Vector<FormDataEntryValue>> entry_list = {});
 
+    JS::ThrowCompletionOr<void> initialize(JS::Realm&) override;
```

Comment:
Nit-pick: Put a `virtual` keyword at the start of this.

Reply:
Thanks, fixed!